src/components/ImageUploader.tsx

```
@@ -0,0 +1,166 @@
+          "relative flex h-[120px] w-[120px] flex-col items-center justify-center rounded-lg transition-colors",
+          !previewUrl && "cursor-pointer hover:bg-gray-50",
+          previewUrl ? "border-none" : "border-2 border-dashed",
+          !previewUrl && isDragging && "border-blue-500 bg-blue-50",
+          !previewUrl && !isDragging && "border-gray-300",
+          !previewUrl && error && "border-red-500",
+          "overflow-hidden",
+        )}
+      >
+        <input
+          ref={inputRef}
+          type="file"
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)